README.md

```diff
@@ -30,6 +30,12 @@ The sort of HealthChecks one could run with Active-Monitor are:
 - verify kube-dns by running DNS lookups on localhost
 - verify KIAM agent by running aws sts get-caller-identity on all available nodes
 
+With the Cluster/Namespace level healtcheck can be run in any namespace provided namespace is already created.
+The `level` in the workflow spec defines at which level the healtcheck runs it can be either namespace or cluster
+when `level` is set to namespace Active-Monitor will create a serviceaccount in the namespace as defined in the workflow spec, it will also create the role and rolebinding with namespace level permissions so that the healthchecks in a namespace can be performed.
```

Comment:
```suggestion
When `level` is set to `Namespace`, Active-Monitor will create a `ServiceAccount` in the namespace as defined in the workflow spec, it will also create the `Role` and `RoleBinding` with namespace level permissions so that the `HealthChecks` in a namespace can be performed.
```